lua/customroles/elementalist/cl_elementalist.lua

```diff
@@ -2,8 +2,15 @@
 local shouldDrawdarkOverlay = false
 local iceOverlay = nil
 local darkOverlay = {
-    ["$pp_colour_brightness"] = 1,
-    ["$pp_colour_colour"] = 1
+    [ "$pp_colour_addr" ] = 0,
```

Comment:
`DrawColorModify` requires full table even though the structure it takes claims there are default values.
This fixes the blind always being full black.